libs/features/mas/web-components/src/merch-card.css.js

```diff
@@ -140,6 +140,7 @@ export const styles = css`
         height: fit-content;
         flex-direction: column;
         width: fit-content;
+        max-width: 140px;
```

Comment:
Try adding `box-sizing: border-box;` to enforce `max-width`, otherwise it will still overlap.
![Screenshot 2024-08-05 at 11 02 47 AM](https://github.com/user-attachments/assets/e1436f05-29b6-4f28-b1c8-8a3606c84917)